tests/contrib/asyncio/test_tracer_safety.py

```diff
@@ -16,8 +16,8 @@
 async def test_get_call_context(tracer):
     tracer.configure(context_provider=DefaultContextProvider())
     # it should return a context even if not attached to the Task
-    ctx = tracer.get_call_context()
-    assert ctx is not None
```

Comment:
Can you explain why `current_trace_context()` returns None when `get_call_context()` returns something?

Reply:
`current_trace_context` returns None because there is no active trace when it is called. 

`get_call_context` returns a context because the implementation was such that there is _always_ a context regardless of whether there is a trace or not.